src/build.d

```diff
@@ -119,6 +121,39 @@ Command-line parameters
     }
     foreach (target; targets.parallel(1))
         target();
+
+    writeln("Success");
```

Comment:
Added this because on rebuilds the program immediately exits, making the user unsure whether anything happened.

Reply:
Yeah sometimes command line tools fall into infinite loop and you don't realize it until the system got saturated...